get-started/04_sharing_app.md

```diff
@@ -58,6 +58,7 @@ If you look at the image below an example **Docker command** can be seen. This c
     ```console
     $ docker tag getting-started YOUR-USER-NAME/getting-started
     ```
+    Learn more about [docker tag](https://docs.docker.com/engine/reference/commandline/tag/).
```

Comment:
Thanks @Lchris22. I just noticed it was an absolute link. Can we change it to a relative link. 

```suggestion
    Learn more about [docker tag](../engine/reference/commandline/tag.md).
```